Makefile

```diff
@@ -45,7 +45,10 @@ ci-travis:
 	@printf "$$(tput bold)* CI: Formatting *$$(tput sgr0)\n"
 	@printf "$$(tput bold)******************$$(tput sgr0)\n"
 	@CI=true ./tools/run_cargo_fmt.sh diff
-	@./tools/check_wildcard_imports.sh
+	@printf "$$(tput bold)******************$$(tput sgr0)\n"
+	@printf "$$(tput bold)* CI: Linting *$$(tput sgr0)\n"
+	@printf "$$(tput bold)******************$$(tput sgr0)\n"
+	@./tools/run_clippy.sh
```

Comment:
In  my opinion, if we're adding clippy to Travis:

 1. There should be a simple command (like `make format`) that people can use to run it locally
 2. It should be clear that people have to run it locally in order to pass travis. The best might be for it to be run automatically for them (but not edit the code, just report errors). Or otherwise, maybe it should be another checkbox on the PR template?

Reply:
My vote would be for `make format` to call `run_clippy.sh`, but I would also be fine with adding a `make lints` or similar and putting it in the PR template.

Reply:
I think breaking it out to `make lints` or even maybe more explicitly `make clippy` makes sense. It's nice to be able to iterate on the piece that's causing you problems.

We already have `make ci` that runs the full CI suite locally. That takes a while to run, however; it's not totally clear what the line should be for what we prompt people on in the PR template.

Maybe `make lints` could run `fmt` and `clippy` and we prompt that. Maybe we add a `make ci-quick` for the fast targets that make sense locally?